src/index.d.ts

```diff
@@ -133,6 +133,7 @@ declare module 'discord-akairo' {
         public compareStreaming(oldMember: GuildMember, newMember: GuildMember): number;
         public embed(data?: object): MessageEmbed;
         public fetchMember(guild: Guild, id: string, cache?: boolean): Promise<GuildMember>;
+        public permissionNames(): string[];
```

Comment:
Import `Permissions` from discord.js on line 8

```suggestion
        public permissionNames(): (keyof Permissions['FLAGS'])[];
```

Reply:
Done!

Reply:
Didn't add the import on line 8

Reply:
Should be done by now!